### PR TITLE
Health check benjamin

### PR DESCRIPTION
**Pull request by galois-dev**

### CODE REVIEW

File: src/utils/healthCheck/healthAudit.ts [hunk 1]
```diff
@@ -0,0 +1,107 @@
+import { useQuery } from '@apollo/react-hooks'
+import { CloudinaryContext } from 'cloudinary-react'
+import { useAuth0 } from './../../lib/react-auth0-spa'
```
Comment (nopestack) [inline src/utils/healthCheck/healthAudit.ts:3]: Please use absolute paths to avoid this

**Review by nopestack — CHANGES_REQUESTED** [review]
Please remove `package-lock.json` as we use yarn and we already have a `yarn.lock` and these would conflict upon deployment.

File: src/utils/healthCheck/healthAudit.ts (hunk 1 above)
Comment (galois-dev) [reply src/utils/healthCheck/healthAudit.ts:3]: I didn't finish any of this code I was told to drop it and leave it as is and go on to something else while leaving the old branch here. Dont read any of this

**Comment by nopestack** [comment]
I'm closing this PR as it's a pretty substantial change that requires attention and we're focusing on other priorities at the moment. We can reopen and revisit this at a later date to follow up this conversation, the branch will remain intact however

**Pull request closed without merging**